<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_device</class>
 <widget class="QDialog" name="show_device">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Show device</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>701</width>
     <height>801</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-50</y>
     <width>681</width>
     <height>811</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>24</number>
    </property>
    <property name="topMargin">
     <number>29</number>
    </property>
    <property name="rightMargin">
     <number>24</number>
    </property>
    <property name="bottomMargin">
     <number>29</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>29</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Назва:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lname">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>нзв</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">  QLineEdit{
	border-radius: 8px;
	border: 2px solid grey;	
}

QLineEdit:focus{
	border: 2px solid #E0DAFF;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">height: 5px;
background-color:black;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>29</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Тип гаджета</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="typeEdit" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QRadioButton" name="laptop">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>110</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold Condensed</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ноутбук</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="phone">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>110</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold Condensed</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Смартфон</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="tv">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>10</y>
           <width>110</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold Condensed</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Телевізор</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="type">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>нзв</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">height: 5px;
background-color:black;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>29</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ціна</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="price">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>нзв</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="priceEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">  QLineEdit{
	border-radius: 8px;
	border: 2px solid grey;	
}

QLineEdit:focus{
	border: 2px solid #E0DAFF;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">height: 5px;
background-color:black;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>29</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Рік випуску</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="year">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>нзв</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yearEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">  QLineEdit{
	border-radius: 8px;
	border: 2px solid grey;	
}

QLineEdit:focus{
	border: 2px solid #E0DAFF;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">height: 5px;
background-color:black;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>29</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Діагональ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="display">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>нзв</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="displayEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">  QLineEdit{
	border-radius: 8px;
	border: 2px solid grey;	
}

QLineEdit:focus{
	border: 2px solid #E0DAFF;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">height: 5px;
background-color:black;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="topMargin">
       <number>29</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Батарея</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="battery">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>нзв</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="batteryEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">  QLineEdit{
	border-radius: 8px;
	border: 2px solid grey;	
}

QLineEdit:focus{
	border: 2px solid #E0DAFF;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #F2F0FF;
	border-radius: 8px;
	border: 2px solid grey;	
}
QPushButton:hover{
	background-color: #E0DAFF;

}
</string>
        </property>
        <property name="text">
         <string>Редагувати</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #F2F0FF;
	border-radius: 8px;
	border: 2px solid grey;	
}
QPushButton:hover{
	background-color: #E0DAFF;

}
</string>
        </property>
        <property name="text">
         <string>Зберегти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #F2F0FF;
	border-radius: 8px;
	border: 2px solid grey;	
}
QPushButton:hover{
	background-color: #E0DAFF;

}
</string>
        </property>
        <property name="text">
         <string>Видалити</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #F4FFFF;
	border-radius: 8px;
	border: 2px solid grey;	
}
QPushButton:hover{
	background-color:  #dbffff;

}
</string>
        </property>
        <property name="text">
         <string>Закрити вікно</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>2</width>
     <height>640</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2</width>
     <height>640</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 40px;
background-color:black;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
